src/state/stake/hooks.ts

```diff
@@ -194,14 +194,13 @@ export function useSyrupInfo(
       chainId
         ? returnSyrupInfo()
             [chainId]?.slice(startIndex, endIndex)
-            .filter((stakingRewardInfo) =>
-              tokenToFilterBy === undefined || tokenToFilterBy === null
-                ? getSearchFiltered(
-                    stakingRewardInfo,
-                    filter ? filter.search : '',
-                  )
-                : tokenToFilterBy.equals(stakingRewardInfo.token) &&
-                  tokenToFilterBy.equals(stakingRewardInfo.token),
+            .filter(
+              (syrupInfo) =>
+                syrupInfo.ending > Math.floor(Date.now() / 1000) &&
```

Comment:
Make the / 1000 more clear e.g method should be like 
GetDateInSeconds etc.

I'm the future times should be named in their stored value e.g. EndingDateInSeconds